core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -1276,4 +1278,13 @@ public static boolean hasMVDataMap(CarbonTable carbonTable) throws IOException {
     }
     return false;
   }
+
+  /**
+   * if format is 'carbondata' or '' or null, return false
+   * else return true.
+   */
+  public boolean isExternalFormat() {
```

Comment:
please optimize it to 'isExternalFormatTable'

Reply:
fixed